src/assets/variables.scss

```diff
@@ -22,15 +22,15 @@
 
 // https://uxplanet.org/7-rules-for-mobile-ui-button-design-e9cf2ea54556
 // recommended is 48px
-// 44px is what we choose and have very good visual-to-usability ratio
-$clickable-area: 44px;
+// 34px is what we choose and have very good visual-to-usability ratio
+$clickable-area: 34px;
```

Comment:
This will break stuff.
Instead everything that uses SCSS variables should be ported to CSS variables.